tests/fixtures/data_fixtures.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from PIL import Image
+import pytest
+
+from fairywren.clip import CLIPFaissIndex
+
+
+@pytest.fixture()
+def test_embeddings():
+    return np.ones((2, 512), dtype=np.float32)
+
+
+@pytest.fixture()
+def test_embeddings_non512():
+    return np.ones((2, 4), dtype=np.float32)
+
+
+@pytest.fixture()
+def test_metadata():
+
+    return ['url1', 'url2']
+
+
+@pytest.fixture()
+def test_metadata_mismatch():
+
+    return ['url1']
+
+
+@pytest.fixture()
+def fairywren_clip_index():
+    wren_index = CLIPFaissIndex(embedding_array=np.ones((2, 512), dtype=np.float32),
+                                id_metadata=['url1', 'url2']
+                                )
```

Comment:
[totally optional] mostly curious, would this work?
```suggestion
def fairywren_clip_index(test_embeddings, test_metadata):
    wren_index = CLIPFaissIndex(embedding_array=test_embeddings,
                                id_metadata=test_metadata
                                )
```

Reply:
yeah I think that technically would have worrrkeeeddddddd????